### PR TITLE
Update go 1.13 revision, add 1.14 and 1.15 versions as tests harnesses

### PR DESCRIPTION
**Pull request by lggomez**
First step related to #408

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -1,21 +1,39 @@
 
+#!/bin/bash
```
Comment (MichaelS11) [inline test.sh:2]: Please remove bash, can call it with sh

File: test.sh [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
 echo "installing go"
 cd /tmp/
-wget -nv https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz
-wget -nv https://dl.google.com/go/go1.12.17.linux-amd64.tar.gz
-wget -nv https://dl.google.com/go/go1.11.13.linux-amd64.tar.gz
-wget -nv https://dl.google.com/go/go1.10.8.linux-amd64.tar.gz
-wget -nv https://dl.google.com/go/go1.9.7.linux-amd64.tar.gz
+
+FILE=go1.15.6.linux-amd64.tar.gz
+[ ! -e "${FILE}" ] && wget -nv https://dl.google.com/go/go1.15.6.linux-amd64.tar.gz
```
Comment (MichaelS11) [inline test.sh:13]: Please either remove file or use it in both places.

File: test.sh [hunk 3]
```diff
@@ -1,21 +1,39 @@
...
+[ ! -e "${FILE}" ] && wget -nv https://dl.google.com/go/go1.12.17.linux-amd64.tar.gz
+FILE=go1.11.13.linux-amd64.tar.gz
+[ ! -e "${FILE}" ] && wget -nv https://dl.google.com/go/go1.11.13.linux-amd64.tar.gz
+FILE=go1.10.8.linux-amd64.tar.gz
+[ ! -e "${FILE}" ] && wget -nv https://dl.google.com/go/go1.10.8.linux-amd64.tar.gz
+FILE=go1.9.7.linux-amd64.tar.gz
+[ ! -e "${FILE}" ] && wget -nv https://dl.google.com/go/go1.9.7.linux-amd64.tar.gz
+
+mkdir -p /usr/local/goFiles1.15.x
+tar -xf /tmp/go1.15.6.linux-amd64.tar.gz
```
Comment (MichaelS11) [inline test.sh:28]: If you are creating a var, you could use it here as well.

File: test_cleanup.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (MichaelS11) [inline test_cleanup.sh:1]: Please remove bash, call with sh

File: test_cleanup.sh [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+rm -rf /usr/local/go1.15.x || true
```
Comment (MichaelS11) [inline test_cleanup.sh:3]: Is || true needed?

File: test_cleanup.sh [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+rm -rf /usr/local/go1.14.x || true
+rm -rf /usr/local/go1.13.x || true
+rm -rf /usr/local/go1.12.x || true
+rm -rf /usr/local/go1.11.x || true
+rm -rf /usr/local/go1.10.x || true
+rm -rf /usr/local/go1.9.x || true
+rm -rf /usr/local/pkg_config || true
```
Comment (MichaelS11) [inline test_cleanup.sh:10]: Add newline at end

**Review by MichaelS11 — CHANGES_REQUESTED** [review]

File: test_cleanup.sh (hunk 5 above)
Comment (lggomez) [reply test_cleanup.sh]: Turns out that -f takes care of errors so in this case it actually isn't

**Comment by lggomez** [comment]
With the given feedback, I refactored the code into functions (including the go test section). First section seems to work, I can't reach the second part as I am not able to startup the database

Do you have a startup.sh script that you can share? I installed oracle 11.2xe via alien on linux mint 20, so I'm not discarding something with my local installation may be wrong

**Comment by MichaelS11** [comment]
Not sure exactly what you are looking for. Did you check out https://github.com/mattn/go-oci8/blob/master/.travis.yml ?

**Review by MichaelS11 — APPROVED** [review]

**Comment by MichaelS11** [comment]
Here is the startup.sh file:
https://github.com/wnameless/docker-oracle-xe-11g/blob/master/assets/startup.sh

**Comment by MichaelS11** [comment]
@lggomez Thank you! :)

**Pull request merged**